fractal_server/app/routes/admin/v2.py

```diff
@@ -307,3 +313,113 @@ async def flag_task_v1_as_v2_compatible(
     await db.close()
 
     return Response(status_code=status.HTTP_200_OK)
+
+
+class TaskV2Minimal(BaseModel):
+
+    id: int
+    name: str
+    type: str
+    command_non_parallel: Optional[str]
+    command_parallel: Optional[str]
+    source: str
+    owner: Optional[str]
+    version: Optional[str]
+
+
+class ProjectUser(BaseModel):
+
+    id: int
+    email: EmailStr
+
+
+class TaskV2Relationship(BaseModel):
+
+    workflow_id: int
+    workflow_name: str
+    project_id: int
+    project_name: str
+    project_users: list[ProjectUser] = Field(default_factory=list)
+
+
+class TaskV2Info(BaseModel):
+
+    task: TaskV2Minimal
+    relationships: list[TaskV2Relationship]
+
+
+@router_admin_v2.get("/task/", response_model=list[TaskV2Info])
+async def query_tasks(
+    id: Optional[int] = None,
+    source: Optional[str] = None,
+    version: Optional[str] = None,
+    name: Optional[str] = None,
+    owner: Optional[str] = None,
+    kind: Optional[Literal["common", "users"]] = None,
+    max_number_of_results: int = 25,
+    user: User = Depends(current_active_superuser),
+    db: AsyncSession = Depends(get_async_db),
+) -> list[TaskV2Info]:
```

Comment:
Let's add a docstring here, especially to document the behavior for each query parameter (exact match vs case-insensitive contains).